### PR TITLE
Refactor parDim to be more readable/intuitive.

### PR DESCRIPTION
**Pull request by npadmana**
Instead of using an `int`, use an `enum`. This also hopefully makes what the code is doing a little more clear.

Tested locally on my laptop with sizes S,W,A,B,C

### CODE REVIEW

**Comment by npadmana** [comment]
@ronawho -- care to review? In particular, does this make this part of the code more clear....?

File: src/DistributedFFT.chpl [hunk 1]
```diff
@@ -188,6 +188,22 @@ prototype module DistributedFFT {
...
+     equal to the number of columns = length of a row. Doing the FFT along a column has a stride
+     equal to the number of columns = length of a row, but idist=1 (corresponding elements of each
+     array are separated by 1).
+
+     We define an enum to switch between these cases.
+     ParDim.Row => do the FFT along the row (contiguous data)
+     ParDim.Column => do the FFT along the column (non-contiguous).
+  */
+  pragma "no doc"
+  enum ParDim { Row, Column };
```
Comment (ronawho) [inline src/DistributedFFT.chpl:205]: Slight preference for `Column` -> `Col`

File: src/DistributedFFT.chpl [hunk 2]
```diff
@@ -272,9 +288,9 @@ prototype module DistributedFFT {
       const myLineSize = zSrc.size*numBytes(T);
 
       // Setup FFTW plans
-      var xPlan = setupBatchPlan(T, ftType, {xDst, zSrc}, parDim=2, signOrKind, FFTW_MEASURE);
-      var yPlan = setupBatchPlan(T, ftType, {ySrc, zSrc}, parDim=2, signOrKind, FFTW_MEASURE);
-      var zPlan = setupPlan(T, ftType, {0..0, zSrc}, parDim=1, 1, signOrKind, FFTW_MEASURE);
+      var xPlan = setupBatchPlan(T, ftType, {xDst, zSrc}, parDim=ParDim.Column, signOrKind, FFTW_MEASURE);
+      var yPlan = setupBatchPlan(T, ftType, {ySrc, zSrc}, parDim=ParDim.Column, signOrKind, FFTW_MEASURE);
+      var zPlan = setupPlan(T, ftType, {0..0, zSrc}, parDim=ParDim.Row, 1, signOrKind, FFTW_MEASURE);
```
Comment (ronawho) [inline src/DistributedFFT.chpl:293]: Nit -- could drop `parDim=` since ParDim is included in the enum name now.

Comment (npadmana) [reply src/DistributedFFT.chpl:293]: Done.

**Comment by npadmana** [comment]
Great, thanks for looking this over!

**Pull request merged**